sapp/ui/frontend/src/Issue.js

```diff
@@ -130,12 +134,62 @@ export type IssueDescription = {
   source_names: $ReadOnlyArray<string>,
   sinks: $ReadOnlyArray<string>,
   sink_names: $ReadOnlyArray<string>,
+  status: string,
   features: $ReadOnlyArray<string>,
   min_trace_length_to_sources: number,
   min_trace_length_to_sinks: number,
   is_new_issue: boolean,
 };
 
+const StatusSelect = (
+  props: $ReadOnly<{
+    issue_id: number,
+    status: string,
+  }>,
+): React$Node => {
+  const allStatuses = [
+    "Uncategorised",
```

Comment:
@abishekvashok What do you think about having a dictionary which maps the status in the same version as the backend (IssueStatus.BAD_PRACTICE bad_practice) with a human readable version. Something like 
{
"bad_practice": "Bad Practice",
...
}
In this way we can show to the user the human readable version but send to the graphQL query the same representation defined in the backend (IssueStatus )
This would avoid having to do all the `status.replace(" ", "_").lower()` conversions between the human readable and the backed IssueStatus representation in the python code which I think can be very error prone

Reply:
Yep makes sense!